plugins/destination/snowflake/client/write.go

```diff
@@ -54,13 +55,20 @@ func (c *Client) WriteTableBatch(ctx context.Context, name string, msgs message.
 	if err := f.Close(); err != nil {
 		return fmt.Errorf("failed to close temp file with last resource %s: %w", f.Name(), err)
 	}
-	sql := fmt.Sprintf(putFileIntoStage, f.Name())
+
+	sql := fmt.Sprintf(putFileIntoStage, escapePath(f.Name()))
+
 	if _, err := c.db.ExecContext(ctx, sql); err != nil {
 		return fmt.Errorf("failed to put file into stage with last resource %s: %w", sql, err)
 	}
-	sql = fmt.Sprintf(copyIntoTable, tableName, path.Base(f.Name()))
+	sql = fmt.Sprintf(copyIntoTable, tableName, escapePath(path.Base(f.Name())))
 	if _, err := c.db.ExecContext(ctx, sql); err != nil {
 		return fmt.Errorf("failed to copy file into table with last resource %s: %w", sql, err)
 	}
 	return err
 }
+
+// escapePath properly escapes the `\` character in window's file paths.
+func escapePath(p string) string {
```

Comment:
why not use https://pkg.go.dev/path/filepath package & funcs from it?

Reply:
Specifically, they do handle this: https://cs.opensource.google/go/go/+/refs/tags/go1.21.1:src/path/filepath/path_windows.go

Reply:
What function do you want me to use?

Reply:
Oh, I missed that it is escape, not unescape